app/views/admin/users/_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= simple_form_for user, url: url do |f| %>
   <%= f.input :name %>
   <%= f.input :email %>
-  <%= f.input :admin, as: :radio_buttons %>
   <%= f.input :password %>
-  <%= f.submit %>
+  <%= f.input :admin, as: :radio_buttons %>
+  <div class="submit"><%= f.submit %></div>
```

Comment:
A suggestion to break these tags to individual lines

Reply:
Done in sha: a2fa585